static/js/settings_org.js

```diff
@@ -248,7 +248,7 @@ exports.extract_property_name = function (elem) {
 
 function get_subsection_property_elements(element) {
     const subsection = $(element).closest('.org-subsection-parent');
-    return subsection.find('.prop-element');
+    return subsection.find('.prop-element').toArray();
```

Comment:
`return Array.from(subsection.find('.prop-element'))` would avoid the unpleasant mocking changes.